src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -148,6 +149,13 @@ export class CategoryFacet extends Component implements IAutoLayoutAdjustableIns
      */
     collapsedByDefault: ComponentOptions.buildBooleanOption({ defaultValue: false, section: 'Filtering', depend: 'enableCollapse' }),
 
+    /**
+     * Whether to notify the [Breadcrumb]{@link Breadcrumb} component when toggling values in the facet.
+     *
+     * **Default:** `true`
```

Comment:
You can remove **Default:** `true`; TypeDoc will generate a tag from the buildBooleanOption argument